snare/cloner.py

```diff
@@ -134,6 +149,8 @@ def _make_filename(self, url):
             content_type = None
             try:
                 response = await session.get(current_url, headers={'Accept': 'text/html'}, timeout=10.0)
+                headers = self.get_headers(response)
+                # TODO might not be needed anymore if we can rely on the header
                 content_type = response.content_type
```

Comment:
If this prq makes sense, I can change the code to rely on the content-type found in the header instead of this one.